VMWare/vmware-esxi/_meta/manifest.yml

```diff
@@ -0,0 +1,8 @@
+uuid: 2b13307b-7439-4973-900a-2b58303cac90
+name: vmware-esxi
+slug: vmware-esxi
+
+description: >-
+  The description of the intake
+
+data_sources:
```

Comment:
Can you add the most suitable datasources from [this list](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/datasources.md)?